askama/src/lib.rs

```diff
@@ -126,42 +132,17 @@ impl<T: Template> DynTemplate for T {
     fn size_hint(&self) -> usize {
         Self::SIZE_HINT
     }
+
+    fn mime_type(&self) -> &'static str {
+        Self::MIME_TYPE
+    }
 }
 
 pub use crate::shared::filters;
 pub use crate::shared::helpers;
 pub use crate::shared::{read_config_file, Error, MarkupDisplay, Result};
 pub use askama_derive::*;
 
-pub mod mime {
```

Comment:
Should stay for semver compat.

Reply:
But the `#[cfg(all(feature = "mime_guess", feature = "mime"))]` could probably be moved to the `mod` instead of each item

Reply:
It's public, so we need to keep it  around for semver-compat reasons. However, we can deprecate all these!

Reply:
Makes sense. Still kinda funny that empty module is available if the feature is disabled, haha